HousingRepairsOnlineApi/Helpers/UpdateOrCancelAppointmentStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace HousingRepairsOnlineApi.Helpers;
+
+public enum UpdateOrCancelAppointmentStatus
+{
+    Unknown,
+    AppointmentUpdated,
```

Comment:
Adding/inserting a new value to this enum has made me think that it would be best to reorder the values to have the non-successful values between `Unknown` and the successful values.
This would allow appending successful values to the end without impact existing usages.